ipn/ipnlocal/local.go

```diff
@@ -988,7 +995,11 @@ func (b *LocalBackend) WhoIs(ipp netip.AddrPort) (n tailcfg.NodeView, u tailcfg.
 	if !ok {
 		var ip netip.Addr
 		if ipp.Port() != 0 {
-			ip, ok = b.sys.ProxyMapper().WhoIsIPPort(ipp)
+			if proto == "" {
+				proto = "tcp"
```

Comment:
I think we should err on the side of caution and check both tcp and udp here if the caller didn't specify it.

Reply:
Done 👍